scripts/repo-build

```diff
@@ -93,3 +93,7 @@ touch --no-dereference --date="@$lastcommitdate" \
     "$repodir"/Packages.stamps
 
 section "Done. Result is in $repodir"
+
+section "Making packages web listing"
+echo "${BASH_SOURCE%/*}"/repo-build-web "$recipesdir" "$repodir"
+"${BASH_SOURCE%/*}"/repo-build-web "$recipesdir" "$repodir"
```

Comment:
I’ve been changing the other occurrences of `${BASH_SOURCE%/*}/` to simply `scripts/` in other scripts lately, because I think it looks confusing and because we only run the scripts from the repo root anyway. If you agree with this, could you please change those two lines to have `scripts/` instead of `${BASH_SOURCE%/*}/`? Otherwise, we can discuss about it and maybe change it back in the other places.